Formula/r/rpm.rb

```diff
@@ -21,32 +21,34 @@ class Rpm < Formula
   depends_on "cmake" => :build
   depends_on "doxygen" => :build
   depends_on "python@3.12" => [:build, :test]
-  depends_on "acl"
-  depends_on "bzip2"
-  depends_on "dbus"
-  depends_on "elfutils"
+
   depends_on "gettext"
   depends_on "libarchive"
-  depends_on "libcap"
   depends_on "libmagic"
-  depends_on :linux
   depends_on "lua"
   depends_on "openssl@3"
   depends_on "pkg-config"
   depends_on "popt"
   depends_on "sqlite"
   depends_on "xz"
-  depends_on "zlib"
+  depends_on "zstd"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
 
-  on_linux do
-    conflicts_with "rpm2cpio", because: "both install `rpm2cpio` binaries"
+  on_macos do
+    depends_on "libomp"
   end
 
+  conflicts_with "rpm2cpio", because: "both install `rpm2cpio` binaries"
```

Comment:
Could we disable this conflicting install instead?

Reply:
this was originally here and in rpm2cpio as well